### PR TITLE
added check_log4j_used()

### PR DESCRIPTION
**Pull request by slrbl**
check_log4shell_used() will check if log4j is present in the system before launching a scan

### CODE REVIEW

**Comment by phantinuss** [comment]
You might want to check systems that have logs of log4j but do not actually run it by themselves. In my opinion the check should be made optional with e.g. a command line flag.

Also there is a dependency on jar. Not every system will have it available in the $PATH. 
`grep -r --include *.[wj]ar "JndiLookup.class" / 2>&1 | grep matches`
should offer a similar result

**Comment by slrbl** [comment]
I added the flag '-c' to make it optional. Also updated the jar checker.

**Comment by Neo23x0** [comment]
Could you add a sentence to the README to explain what that flag does? 

https://github.com/Neo23x0/log4shell-detector/blob/main/README.md#special-flags

**Comment by slrbl** [comment]
done

File: log4shell-detector.py [hunk 1]
```diff
@@ -40,6 +52,33 @@ def evaluate_log_paths():
     parser.add_argument('--quick', action='store_true', help="Skip log lines that don't contain a 2021 or 2022 time stamp")
     parser.add_argument('--debug', action='store_true', help='Debug output')
     parser.add_argument('--summary', action='store_true', help='Show summary only')
+    parser.add_argument('--check_usage', '-c',action='store_true', help='Check if log4j is being used before launching the scan')
+
+    args = parser.parse_args()
+    
+    print("     __             ____ ______       ____  ___      __          __          ")
```
Comment (Neo23x0) [inline log4shell-detector.py]: Why is that banner two times in the script? 
Something got mixed up.

Comment (slrbl) [reply log4shell-detector.py]: Fixed

**Pull request merged**